src/stores/Tabs/tab_filter.js

```diff
@@ -53,7 +54,11 @@ export const TabFilter = types
     },
 
     get component() {
-      return Filters[self.filter.currentType] ?? Filters.String;
+      const operatorList = Filters[self.filter.currentType] ?? Filters.String;
+
+      return (isFF(FF_LOPS_E_3) && getRoot(self)?.SDK?.type === "DE") ? operatorList.filter((op) => {
```

Comment:
Can we extract this operations filter to a shared function, and maybe something like an exclusion list object lookup, something like:

```tsx
// another file
const EXCLUDED_FILTER_OPERATIONS = {
    "DE": ["contains", "not_contains", "regex"]
}

export const allowedFilterOperations = (operationsList: Array<{ key: string; label: string; valueType: string }>, context = "") => {
   return (context in EXCLUDED_FILTER_OPERATIONS) ? operationsList.filter(op => !EXCLUDED_FILTER_OPERATIONS[context].includes(op.key)) : operationsList;
}

// and then in usage

allowedFilterOperations(operationsList, isFF(FF_LOPS_E_3) && getRoot(self)?.SDK?.type === "DE"))
```

Reply:
This way we don't even need the isFF or anything like that, also we remove the usage of vendor specifics.

Reply:
oh interesting, yeah i like this approach better - will do